ISlice.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+type Slice struct {
+	// 1 шаг
+	pointer []int
+	len     int
+	cap     int
+}
+
+func (s *Slice) Init(length, capacity int) *Slice {
```

Comment:
make метод разве вызывается у slice?
ты же юзаешь arr := make([]int, 0, 10) - а не через точку. значит метод не должен принадлежать структуре

```suggestion
func Init(length, capacity int) *Slice {
```